wiki/Multiplayer/en.md

```diff
@@ -1,451 +1,419 @@
-		-   You will be connected to \#lobby by default.
-	-   Check out the players logged into lobby by looking at the right of the screen.
-	-   Select and playing your jukebox songs at the upper-right of the screen.
+_This is a (waiting) Multiplayer Lobby._
+
+It will show all the available Match Setup in the _Bancho_ server.
+Sort out by game mode or fine-tune the search by filtering or searching for a player's name.
+[Chat Console][Chat Console wikilink] will be available at the screen and connected to \#lobby by default.
+
+Each row contains a Match Setup with its game mode symbol, game mode name (Team Mode), number of players inside/number of players allowed, rank difference from highest to lowest at the left section; large host profile image and smaller players profile image in the Match Setup along with the Match Setup name and the beatmap's name \[difficulty\] to be played for this match at the right section.
+
+Click on the row to enter the Match Setup.
+If the game mode symbol contains a lock, clicking on it will prompt a password request; fill in the Match Setup password correctly to enter the locked Match Setup.
+
+A new room can be created by pressing the _New Game_ button, or _Quick Join_ a Match Setup by randomly join a room based on available beatmap.
+
+Click on the _Back to Menu_ button to return to the Main Menu.
 
 ## New game
+
 ![New Game setup screen][Multi_new_game image]
 
+_Setting up a new game_
+
 Title | Description
 :---: | :---
-**Game Name** | **Name of the "room"**. You can use foreign languages (Unicode). <br> Enable the "Require Password to join" button to make your "room" private.
-**Password** | Will only appear if the button is enabled. **The Password to enter the room.** <br> Your "room" sign will be shown with a lock.
-**Max players** | **The number of players that can join your "room".** Default to "8 players" with range from 2-16 players, including self as a player, and can be adjusted later by locking/unlocking the slots.
+**Game Name**   | **Name of the room.** Default to {accountname}'s game. Foreign characters allowed (parsed in Unicode). Enable the Require Password to join button to make the Match Setup private.
+**Password**    | **Password to enter the room.** Only appears if the Require Password to join button was enabled. Match Setup game mode symbol will be shown with a lock.
+**Max players** | **Total number of players allowed to join the Match Setup.** Default to 8 players, with range from 2 to a total of 16 players, including the player as host. It can be adjusted later by locking/unlocking the placements.
 
-After you are done, you can press the "Start Game" to create the "room" with you as the host of the room. You will be directed to **Match Setup**
+Press the `1. Start Game` button to create the room with the player as the host of the room and the beatmap playing in the jukebox as the match's beatmap difficulty.
+
+_osu!_ will be directed to **Match Setup**.
```

Comment:
well `_osu!_` will be directed to **Match Setup**... but who will be directed, osu! itself or the person making the room?

----

^I may have seen a couple of instances like this where the noun "you" has been skipped. You may want to consider that this could confuse whomever is reading it :/